### ISSUE TITLE
Field with callable default value generates an invalid schema

### ISSUE THREAD

**Issue by tiltec** [axnsan12/drf-yasg#25]
Hi @axnsan12, cool project! I tried it out in https://github.com/yunity/karrot-backend and stumbled upon this error:

`TypeError: Object of type 'CurrentUserDefault' is not JSON serializable`
([full stacktrace](https://gist.github.com/tiltec/d4e1f57fec7c76f1b7780ddd71e333ff))

One of our serializers uses DRFs [CurrentUserDefault](http://www.django-rest-framework.org/api-guide/validators/#currentuserdefault) like this:

```py
class FeedbackSerializer(serializers.ModelSerializer):
    class Meta:
        model = FeedbackModel
        fields = ['given_by']
        extra_kwargs = {'given_by': {'default': serializers.CurrentUserDefault()}}
```

When enabling validation, flex throws this error:

```py
flex.exceptions.ValidationError: 'definitions':
    - 'Feedback':
        - 'properties':
            - 'properties':
                - 'given_by':
                    - 'default':
                        - "The value of `default` must be of one of the declared types for the schema. `CurrentUserDefault()` is not one of `['string']`"
    - 'User':
        - 'default':
            - "The value of `default` must be of one of the declared types for the schema. `CurrentUserDefault()` is not one of `['object']`"
```

Is it feasible to add support for CurrentUserDefault into `drf-yasg`?

**Comment by tiltec** [comment]
Oh, just noticed #24, so I'll close this issue again.

**Comment by axnsan12** [comment]
If it does throw an error then this issue does indeed need to be handled. As said in#24, it cannot be handled by making it *work*, but one would at least expect it to not crash and just not populate the default value.

I'll look into it.

**Comment by axnsan12** [comment]
Fixed and pushed 1.0.6.

Can you check if that works for your case?

**Comment by tiltec** [comment]
It works great ✨ Thank you!
